metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/powerbi_api.py

```diff
@@ -47,6 +50,13 @@ def __init__(self, config: PowerBiDashboardSourceConfig) -> None:
             tenant_id=self.__config.tenant_id,
         )
 
+        self.__profiling_resolver = PowerBiDatasetProfilingResolver(
```

Comment:
There are only two way to get the metadata from PowerBI either using regular API or Admin API. You can clearly see this segregation in PowerBI Rest API documentation: https://learn.microsoft.com/en-us/rest/api/power-bi/.

As per doc it looks like it comes under regular-api, so please add profile_dataset in base resolver  and provide implementation in both regular and admin resolver (here it is just pass). Invoke the  powerbi_profiler.py from regular api's profile_dataset  method